jmh-core/src/test/java/org/openjdk/jmh/util/TestListStatistics.java

```diff
@@ -240,17 +240,17 @@ public strictfp void testEmpty() {
         Statistics s = new ListStatistics();
 
         Assert.assertEquals(0, s.getN());
-        Assert.assertEquals(Double.NaN, s.getSum());
-        Assert.assertEquals(Double.NaN, s.getMin());
-        Assert.assertEquals(Double.NaN, s.getMax());
-        Assert.assertEquals(Double.NaN, s.getMean());
-        Assert.assertEquals(Double.NaN, s.getMeanErrorAt(0.5));
-        Assert.assertEquals(Double.NaN, s.getVariance());
-        Assert.assertEquals(Double.NaN, s.getStandardDeviation());
-        Assert.assertEquals(Double.NaN, s.getConfidenceIntervalAt(0.50)[0]);
-        Assert.assertEquals(Double.NaN, s.getConfidenceIntervalAt(0.50)[1]);
-        Assert.assertEquals(Double.NaN, s.getPercentile(0));
-        Assert.assertEquals(Double.NaN, s.getPercentile(100));
+        Assert.assertEquals(Double.NaN, s.getSum(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getMin(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getMax(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getMean(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getMeanErrorAt(0.5), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getVariance(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getStandardDeviation(), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getConfidenceIntervalAt(0.50)[0], 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getConfidenceIntervalAt(0.50)[1], 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getPercentile(0), 0.0000001);
+        Assert.assertEquals(Double.NaN, s.getPercentile(100), 0.0000001);
```

Comment:
Please introduce the `static final double ACCURACY = 0.0000001`, and use it?

Reply:
Done in all places.